### ISSUE TITLE
Add support for php 7.1

### ISSUE THREAD

**Issue by achekulaev** [docksal/docksal#300]
Acquia uses php 7.1. There is a need to add support for php 7.1

**Comment by ArtuGit** [comment]
Has it been already implemented?

**Comment by achekulaev** [comment]
@lmakarov this thing becomes must have...

<img width="835" alt="composer json - sbd-master - projects sbd-master 2017-12-15 15-07-01" src="https://user-images.githubusercontent.com/402007/34058620-a8ef6600-e1a9-11e7-80aa-64a0a62f77a9.png">

**Comment by lmakarov** [comment]
Edge versions of the new 2.0 images are here! 

```
$ fin image registry docksal/cli | grep 'edge.*\.'
docksal/cli:edge-php7.2
docksal/cli:edge-php5.6
docksal/cli:edge-php7.1
docksal/cli:edge-php7.0
```

Feel free to start using them and report any issues in this repo. We will be releasing them and including in the standard stacks with the next Docksal minor release.

**Comment by tamnv** [comment]
@lmakarov 

Thank you!!!

**Comment by lmakarov** [comment]
docksal/cli 2.0 images will be released with Docksal 1.7.0 this month.
